tasks/create-advisory-internal-request/README.md

```diff
@@ -16,6 +16,10 @@ the ReleasePlanAdmission and Application from the Snapshot are also used. The ad
 | synchronously            | Whether the task should wait for InternalRequests to complete                             | Yes      | true                        |
 | pipelineRunUid           | The uid of the current pipelineRun. Used as a label value when creating internal requests | No       |                             |
 
+## Changes in 2.1.0
+- The advisory_url is reported as task result
+  - If the advisory was not created, the result will instead be the empty string
```

Comment:
The taskrun will fail in that case anyway, so I wonder if it even matters. But probably ok.

Reply:
The internal request sets the result as the empty string if the IR failed to create the advisory. That is just for explaining the behavior

Reply:
Yeah, ok, I realized that when reviewing the other PR :)